src/element.ts

```diff
@@ -169,6 +170,8 @@ export class Element {
    * Note that StaveNote calls setGroupStyle() when setStyle() is called.
    */
   addChildElement(child: Element): this {
```

Comment:
It was introduced with this commit:
https://github.com/vexflow/vexflow/commit/da383af131afc9995b63effdde67831d5e5999d5

There is no other addChild in our VexFlow repository. 

Could it be simplified by renaming it to addChild instead of addChildElement?

Reply:
done